controller/network/options.go

```diff
@@ -27,11 +28,15 @@ type Options struct {
 		RerouteFraction float32
 		RerouteCap      uint32
 	}
+	TerminationTimeout time.Duration
+	RouteTimeout       time.Duration
 }
 
```

Comment:
I've been using time.ParseDuration(string), which allows specifying durations as 1s, 5m, etc. 
In this case, not sure it's worth it, since seconds is a very reasonable unit for the measure. Does let you use shorter names, though :)

Reply:
That makes sense. But I'd want to change `cycleSeconds` to match, and I don't want to introduce a breaking change to that right this second.

I feel like there's a pretty big refactoring coming up of the controller once we get to this next chunk of work. We should definitely hit it then.